sklearn/preprocessing/data.py

```diff
@@ -1767,23 +1770,25 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
         Will return sparse matrix if set True else will return an array.
 
     handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown categorical feature is
-        present during transform.
+
+        - 'ignore': Ignore all unknown feature values.
+        - 'error': Raise an error when the value of a feature is unseen during
+          `fit` and out of range of values seen during `fit`.
+        - 'error-strict': Raise an error when the value of a feature is unseen
+          during`fit`.
+
+    copy : bool, default=True
+        If unset, `X` maybe modified in space.
 
     Attributes
     ----------
-    active_features_ : array
-        Indices for active features, meaning values that actually occur
-        in the training set. Only available when n_values is ``'auto'``.
+    feature_index_range_ : array, shape [n_feature, 2]
```

Comment:
I think I prefer the old format, but could be persuaded. I'm not sure, for instance, how non categorical features are encoded in the incumbent `feature_indices_`. As for names: `feature_spans_`? `feature_slices_`? I'd really like to not say "range"...